Resources/Prototypes/Recipes/Reactions/food.yml

```diff
@@ -87,6 +87,19 @@
     - !type:CreateEntityReactionEffect
       entity: FoodCakeBatter
 
+- type: reaction
+  id: CreateTofu
+  impact: Low
+  quantized: true
+  reactants:
+    MilkSoy:
+      amount: 10
```

Comment:
Curdling is a lossy process. Cheese wheel takes 40u milk into 15u of food and 3 slices. Tofu should probably take somewhere around that, anywhere from 30 to 40u soy milk into one tofu block.

Reply:
I feel like 40 units bit too much, so let's try 30 soy milk units